libs/backendless.js

```diff
@@ -1923,7 +1923,18 @@
             } else {
                 var user = Backendless.UserService.getCurrentUser();
 
-                return !!user;
+                if (async) {
+                    //if async need to put it to the end of the stack
+                    setTimeout(function() {
+                        if (!!user) {
+                            responder.success(user);
```

Comment:
I think that success method will not be called, because if we have no `user-token` that's mean the user is not valid, right?